nanofi/src/api/nanofi.cpp

```diff
@@ -131,20 +131,26 @@ nifi_instance *create_instance_repo(const char *url, nifi_port *port, const char
   return instance;
 }
 
+nifi_instance * acquire_standalone_instance () {
+  if (standalone_instance == nullptr) {
+    nifi_port port;
+    auto port_str = utils::IdGenerator::getIdGenerator()->generate().to_string();
+    port.port_id = const_cast<char *>(port_str.c_str());
```

Comment:
We could initialize the port here like
```
nifi_port port{const_cast<char *>(port_str.c_str())};
```

Reply:
changed in [6e43953](https://github.com/martinzink/nifi-minifi-cpp/commit/6e4395376e34445296946998d0110de92b24511e)